### DISCUSSION TITLE
Welcome to cosmic Discussions!

### DISCUSSION THREAD

**Discussion by Ari122003** [Ari122003/cosmic#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ari122003** [comment]
edit the home page

**Reply by Ari122003** [reply]
done

**Comment by RK2203** [comment]
Work --- Shuttle page design (TODO) .

**Reply by RK2203** [reply]
BASIC CHANGES MADE
